charts/kueue/values.yaml

```diff
@@ -150,18 +150,15 @@ webhookService:
       protocol: TCP
       targetPort: 9443
   type: ClusterIP
-
```

Comment:
I'd revert the formatting changes.

Reply:
I was hesitant about that ... it seems the `make artifacts` re-formatted it.
reverting

Reply:
Feel free to send it in a follow up then